vuln/npm/399.json

```diff
@@ -0,0 +1,21 @@
+{
+  "id": "399",
+  "title": "Command Injection - Generic",
+  "overview": "`whereis` concatenates unsanitized input into exec() command",
+  "created_at": "2018-02-25",
+  "updated_at": "2018-03-28",
+  "publish_date": "2018-03-28",
+  "author": "chalker (https://github.com/ChALkeR)",
```

Comment:
Could you use my full name for consistency, like in https://github.com/nodejs/security-wg/blob/master/vuln/npm/394.json#L4 ?